concent_api/core/views.py

```diff
@@ -1220,6 +1225,632 @@ def store_message_and_message_status(
+            reject_report_computed_task = reject_report_computed_task,
+        )
+        mark_message_as_delivered_and_log(database_message, response_to_client)
+        return response_to_client
+
+    elif database_message.response_type == PendingResponse.ResponseType.ForceSubtaskResultsSettled.name:  # pylint: disable=no-member
+        task_to_compute = deserialize_message(database_message.subtask.task_to_compute.data.tobytes())
+        response_to_client = message.concents.SubtaskResultsSettled(
+            task_to_compute = task_to_compute,
+        )
+        mark_message_as_delivered_and_log(database_message, response_to_client)
+        return response_to_client
+
+    elif database_message.response_type == PendingResponse.ResponseType.VerdictReportComputedTask.name:  # pylint: disable=no-member
+        ack_report_computed_task = message.concents.AckReportComputedTask(
+            task_to_compute = deserialize_message(database_message.subtask.task_to_compute.data.tobytes()),
+            subtask_id      = database_message.subtask.subtask_id,
+        )
+        report_computed_task     = deserialize_message(database_message.subtask.report_computed_task.data.tobytes())
+        response_to_client = message.concents.VerdictReportComputedTask(
+            ack_report_computed_task    = ack_report_computed_task,
+            force_report_computed_task  = message.concents.ForceReportComputedTask(
+                report_computed_task = report_computed_task,
+            ),
+        )
+        mark_message_as_delivered_and_log(database_message, response_to_client)
+        return response_to_client
+
+    elif database_message.response_type == PendingResponse.ResponseType.ForceGetTaskResultRejected.name:  # pylint: disable=no-member
+        report_computed_task = deserialize_message(database_message.subtask.report_computed_task.data.tobytes())
+        response_to_client = message.concents.ForceGetTaskResultRejected(
+            force_get_task_result = message.concents.ForceGetTaskResult(
+                report_computed_task = report_computed_task,
+            )
+        )
+        mark_message_as_delivered_and_log(database_message, response_to_client)
+        return response_to_client
+
+    elif database_message.response_type == PendingResponse.ResponseType.ForceGetTaskResultFailed.name:  # pylint: disable=no-member
+        task_to_compute = deserialize_message(database_message.subtask.task_to_compute.data.tobytes())
+        response_to_client = message.concents.ForceGetTaskResultFailed(
+            task_to_compute = task_to_compute,
+        )
+        mark_message_as_delivered_and_log(database_message, response_to_client)
+        return response_to_client
+
+    elif (
+        database_message.response_type in [
+            PendingResponse.ResponseType.ForceGetTaskResultUpload.name,  # pylint: disable=no-member
+            PendingResponse.ResponseType.ForceGetTaskResult.name,  # pylint: disable=no-member
```

Comment:
According to the blueprint `ForceGetTaskResult` is not a valid response type.

Reply:
Done